src/data/repositories/ProjectD2Repository.ts

```diff
@@ -23,32 +27,8 @@ export class ProjectD2Repository implements ProjectRepository {
 
     getList(): FutureData<Project[]> {
         return this.getCategories().flatMap(categories => {
-            return apiToFuture(
-                this.api.models.categoryOptions.get({
-                    fields: {
-                        id: true,
-                        displayName: true,
-                        startDate: true,
-                        endDate: true,
-                        lastUpdated: true,
-                    },
-                    filter: { "categories.code": { eq: categories.project.code } },
-                    order: "displayName:asc",
-                    paging: false,
-                })
-            ).map(d2Response => {
-                return d2Response.objects.map((d2CategoryOption): Project => {
-                    return Project.build({
-                        dataSets: [],
-                        id: d2CategoryOption.id,
-                        name: d2CategoryOption.displayName,
-                        lastUpdated: d2CategoryOption.lastUpdated,
-                        isOpen: this.isProjectOpen(
-                            d2CategoryOption.startDate,
-                            d2CategoryOption.endDate
-                        ),
-                    });
-                });
+            return this.getCategoryOptionsByCode(categories.project.code).map(d2Response => {
```

Comment:
What we typically do here is make getCategoryOptionsByCode return what you really need (and what the name states), the list of categoryOptions, no need for the full response.